include/links-domain.php

```diff
@@ -107,12 +107,8 @@ public function get_hosts() {
 			// The function idn_to_ascii() is much faster than the WordPress method.
 			if ( function_exists( 'idn_to_ascii' ) && defined( 'INTL_IDNA_VARIANT_UTS46' ) ) {
 				$hosts[ $lang ] = idn_to_ascii( $host, 0, INTL_IDNA_VARIANT_UTS46 );
-			} elseif ( class_exists( 'WpOrg\Requests\IdnaEncoder' ) ) {
-				// Since WP 6.2.
-				$hosts[ $lang ] = \WpOrg\Requests\IdnaEncoder::encode( $host );
 			} else {
-				// Backward compatibility with WP < 6.2.
-				$hosts[ $lang ] = Requests_IDNAEncoder::encode( $host );
+				$hosts[ $lang ] = \WpOrg\Requests\IdnaEncoder::encode( $host ); // Since WP 6.2.
```

Comment:
```suggestion
				$hosts[ $lang ] = (string) \WpOrg\Requests\IdnaEncoder::encode( $host ); // Since WP 6.2.
```
According to the doc.

Reply:
I'm not against this, but is it useful? According to the doc, we already have a string https://github.com/WordPress/wordpress-develop/blob/6.6.2/src/wp-includes/Requests/src/IdnaEncoder.php#L58

Reply:
I've suggested that because of the following error:
```
Method PLL_Links_Domain::get_hosts() should return array<string> but returns array<string|false>.
```
That's weird, could `wordpress-phpstan` have a different return type for it?